src/util/Permissions.js

```diff
@@ -32,14 +32,28 @@ class Permissions {
 
   /**
    * Gets all given permissions that are missing from the bitfield.
-   * @param {PermissionResolvable[]} permissions Permissions to check for
+   * @param {PermissionResolvable[]|Permissions} permissions Permissions to check for
    * @param {boolean} [checkAdmin=true] Whether to allow the administrator permission to override
    * @returns {PermissionResolvable[]}
    */
   missing(permissions, checkAdmin = true) {
+    if (permissions instanceof Permissions) return this.missing(permissions.toArray());
```

Comment:
You should pass checkAdmin to your toArray(), for the case someone passes false as checkAdmin to missing. As well as passing it to return this.missing()...